docs/router-components/browser-router.md

```diff
@@ -27,14 +27,15 @@ A `<BrowserRouter>` stores the current location in the browser's address bar usi
 
 ```tsx
 import * as React from "react";
-import * as ReactDOM from "react-dom";
+import {createRoot} from 'react-dom/client';
```

Comment:
```suggestion
import { createRoot } from 'react-dom/client';
```
Just keeping the style consistent

Reply:
Of course, excuse me for that, I'm currently learning React and Javascript in general, so It's my first official PR, but yes you're right I missed that point. Thank for your correction.